mahmoodhamdi/selaty
# Integrate flutter_screenutil and Refactor Views for Improved Responsiveness

### Summary
This PR integrates `flutter_screenutil` into several views to enhance responsiveness and adapt the UI to various screen sizes. The following views have been updated:

- OnboardingView
- AuthView
- LoginView
- RegisterView
- ResetPasswordView
- OtpView
- NewPasswordView
- PasswordChangedSuccessfullyView

### Changes Made
- Added `flutter_screenutil` to project dependencies.
- Replaced hard-coded dimensions with `ScreenUtil` units for padding, margin, font sizes, and other UI elements.
- Ensured that all views adjust dynamically based on screen size and orientation.

### Reason for Changes
By using `flutter_screenutil`, we ensure that the app maintains a consistent and user-friendly layout across various devices and screen sizes. This improves the overall user experience and ensures better visual alignment.

### How to Test
1. Run the app on multiple devices and emulators with different screen sizes and orientations.
2. Verify that the UI elements (such as padding, font sizes, and button dimensions) scale appropriately.
3. Ensure that all views appear visually consistent and functional across devices.

Please review the changes and merge the PR if everything looks good.
